meta-luneos/recipes-qt/qt6/qtbase_git.bbappend

```diff
@@ -106,6 +106,11 @@ SRC_URI:append = " \
     file://9901-Disable-Faux-bolding-in-Qts-FreeType-FontEngine.patch;maxver=6.3.0 \
 "
 
+# Upstream-Status: Inappropriate
+SRC_URI:append:hammerhead-halium = " file://0004-Patch-shaders-in-QRhiGles2-compileShader-to-fix-old-.patch"
+SRC_URI:append:tenderloin-halium = " file://0004-Patch-shaders-in-QRhiGles2-compileShader-to-fix-old-.patch"
+SRC_URI:append:mako = " file://0004-Patch-shaders-in-QRhiGles2-compileShader-to-fix-old-.patch"
```

Comment:
This makes qtbase and everything it belongs on it effectively MACHINE_ARCH.

Reply:
Would
``` SRC_URI:append:cortexa8t2hf-neon-halium ```
be the right approach?

Reply:
TUNE_PKGARCH (cortexa8t2hf-neon-halium) isn't an OVERRIDE:
```
# $OVERRIDES [4 operations]
#   set /OE/build/luneos-kirkstone/webos-ports/openembedded-core/meta/conf/bitbake.conf:801
#     "${TARGET_OS}:${TRANSLATED_TARGET_ARCH}:pn-${PN}:${MACHINEOVERRIDES}:${DISTROOVERRIDES}:${CLASSOVERRIDE}${LIBCOVERRIDE}:forcevariable"
#   append /OE/build/luneos-kirkstone/webos-ports/meta-webos-ports/meta-luneos/conf/distro/include/luneos.inc:106
#     [vardepsexclude] "WEBOS_EXTRA_MACHINEOVERRIDES"
#   set /OE/build/luneos-kirkstone/webos-ports/meta-webos-ports/meta-luneos/conf/distro/include/luneos.inc:206
#     [vardepsexclude] "MACHINE MACHINE_CLASS"
#   set /OE/build/luneos-kirkstone/webos-ports/openembedded-core/meta/conf/documentation.conf:304
#     [doc] "BitBake uses OVERRIDES to control what variables are overridden after BitBake parses recipes and configuration files."
# pre-expansion value:
#   "${TARGET_OS}:${TRANSLATED_TARGET_ARCH}:pn-${PN}:${MACHINEOVERRIDES}:${DISTROOVERRIDES}:${CLASSOVERRIDE}${LIBCOVERRIDE}:forcevariable"
OVERRIDES="linux-gnueabi:arm:pn-qtbase:armv7a:halium:hammerhead-halium:hammerhead-halium:luneos:class-target:libc-glibc:forcevariable"
```

Can it be applied to all halium builds?

Reply:
I applied it on armv7a , it makes more sense I think. But as this patch breaks tissot display, it needs some rework, so let's wait before merging anything